data/engine.py

```diff
@@ -0,0 +1,118 @@
+import sys
+import logging
+import os
+import heapq
+
+# The location of your spark root directory
+spark_home = '/projects/spark'
```

Comment:
Yes, that's correct. Although I think these environment variables should be set in the Makefile instead of in the python script.